index.html

```diff
@@ -30,14 +31,100 @@ <h2>I am a Software Developer</h2>
             </p>
     
             <ul class="social_media">
-                <li><a href=""><i class="fa fa-github fa-2x"></i></a></li>
+                <li><a href="#"><i class="fa fa-github fa-2x"></i></a></li>
                 <li><a href="#"><i class="fa fa-linkedin fa-2x"></i></a></li>
                 <li><a href="#"><i class="fa fa-angelist fa-2x"></i></a></li>
                 <li><a href="#"><i class="fa fa-twitter fa-2x"></i></a></li>
                 <li><a href="#"><i class="fa fa-medium fa-2x"></i></a></li>
             </ul>
-    
         </section>
+        <!-- Branch 2: Work and About me -->
+        <section class="main_middle_section">
+            <h3>My Recent Works</h3>
+
+            <div class="title_line"></div>
+
+            <div class="main_articles_container">
+                <article class="multi_post_card">
+                    <img class="multi_post_image" src="./assets/multi_post_image.svg" alt="stories image">
+                    <h4>Multi-Post Stories</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <div class="techs_used">
+                        <p>css</p>
+                        <p>html</p>
+                        <p>bootsrap</p>
+                        <p>Ruby</p>
+                    </div>
```

Comment:
- [ ] Kindly use `ul` and `li` tags for this subsection instead of `div` and `p` tags. The usage of these tags makes them more semantic when read by other developers and they can instantly know what you are creating. Kindly implement this change for the language badges on all cards. 👍